libtransmission/rpc-server.cc

```diff
@@ -513,11 +509,7 @@ void handle_request(struct evhttp_request* req, void* arg)
 
     auto* server = static_cast<tr_rpc_server*>(arg);
 
-#if LIBEVENT_VERSION_NUMBER >= 0x02020001
-    char const* remote_host = nullptr;
-#else
     char* remote_host = nullptr;
-#endif
     auto remote_port = ev_uint16_t{};
```

Comment:
Please keep those lines. It will be much easier to switch branches without having to redo this.